packages/okta-react/src/SecureRoute.js

```diff
@@ -14,6 +14,32 @@ import React, { Component } from 'react';
 import { Route } from 'react-router';
 import withAuth from './withAuth';
 
+
```

Comment:
@jmelberg-okta Could you please merge this into master. Then I can also create a PR for https://github.com/okta/okta-oidc-js/issues/70 . As that also deals with `SecureRoute` I'd like to avoid merge conflicts later on.